### PR TITLE
Footer done

### LINKED ISSUE #17: FR: Footer Block for RoadChoice.com
image

### PR DESCRIPTION
**Pull request by SantiagoHomps-NC**
Fix #17

Test URLs:
- Before: https://main--vg-roadchoice-com--hlxsites.hlx.page/
- After: https://17-footer--vg-roadchoice-com--hlxsites.hlx.page/

### CODE REVIEW

**Comment by manuel-vara** [comment]
For the internal urls, could you create the links with the absolute URL, please? 
i.e. Pointing to https://main--vg-roadchoice-com--hlxsites.hlx.page/about instead of /about

**Review by manuel-vara — CHANGES_REQUESTED** [review]
Please update internal links to use absolute url:  https://main--vg-roadchoice-com--hlxsites.hlx.page/

**Comment by SantiagoHomps-NC** [comment]
> Please update internal links to use absolute url: https://main--vg-roadchoice-com--hlxsites.hlx.page/

DONE 🚀

File: blocks/footer/footer.js [hunk 1]
```diff
@@ -20,6 +20,27 @@ export default async function decorate(block) {
...
+    const [social, internal, privacy] = children;
+
+    social.classList.add('footer-social-section');
+    internal.classList.add('footer-internal-section');
+    privacy.classList.add('footer-privacy-section');
+
+    social.querySelectorAll('a').forEach((link) => {
+      link.target = '_blank';
+      const linkText = link.innerText.toLowerCase();
+      const icon = document.createElement('i');
```
Comment (jonatan-lledo-netcentric) [inline blocks/footer/footer.js:34]: this can be updated after my Header's PR because I add an updated version of _createElement()_ from mack

**Review by jonatan-lledo-netcentric — APPROVED** [review]

File: blocks/footer/footer.js (hunk 1 above)
Comment (SantiagoHomps-NC) [reply blocks/footer/footer.js:34]: I can change it afterwards

**Review by manuel-vara — APPROVED** [review]

**Pull request merged**